### ISSUE TITLE
JAB remediation verifications due by November 29th

### ISSUE THREAD

**Issue by mogul** [18F/cg-product#477]
Fallout from the JAB Q&A sessions yet to do:
- [x] TR587-AU-5 mitigation (shutdown when logging fails) https://github.com/18F/cg-product/issues/476

Documentation work yet to do:
- [x] Small SSP updates to 1-11, AC, IA https://github.com/18F/cg-product/issues/421
- [x] Small SSP updates to AU, CA, CM, CP, MA, MP, RA, PS https://github.com/18F/cg-product/issues/485
- [x] Small SSP updates to AT, IR, PE, PL, SA, SC, SI https://github.com/18F/cg-product/issues/486
- [x] Document remediation plan and milestones for everything in the POA&M https://github.com/18F/cg-product/issues/422

Technical work awaiting verification from 3PAO
- [x] TR131-IA-12(11): SecureAuth tokens via e-mail https://github.com/18F/cg-product/issues/431 
  - Update: we'll fix this as part of working on https://github.com/18F/cg-product/issues/544 very soon
- [x] V4-AC2-12 atypical usage monitoring
- [x] Container security tests (with SAR updated)
- [x] AU-9 ELK log tampering is not possible/results in alerts
- [x] KOSI-3 code scanning at repos
- [ ] TR10 Verify that staging is adequately separated
  - Update: we should check with 3PAO on this one (12/5 comment worksheet is unclear)
- [ ] TR11 Verify that Nessus is in a separate VPC
  - Update: we should check with 3PAO on this one (12/5 comment worksheet is unclear)
- [ ] TR534, TRV531 Test with SecureAuth
  - Update: 534 is not a testing problem - it's probably an OR now. Not tracked.
  - Update: 531 is confusing - according to the latest SRTM, it looks like this is resolved - why is it still an "active" comment? I think testing is done for this one. Check with 3PAO.
- [ ] TR1023 Testing Riemann and InfluxDB
  - Update: looks like according to the latest comment workbook, this is resolved - why still active? Check with 3PAO.

Documentation work awaiting verification from 3PAO
- [x] KO19-IA5: Fugacious defaults are shorter/fewer
- [x] V66-SC-21 - Document alternative implementation on DNSSEC
  - Unclear status - it's listed as "completed" in the workbook. Check with 3PAO.
- [x] V3-AC2 account management
- [x] V9-CA7 continuous monitoring
- [x] V10-CP4 contingency plan testing
- [x] V11-CP4(1) contingency plan testing
- [ ] V12-IA5(6) authenticator management - tracked at https://github.com/18F/cg-product/issues/399
  - Unclear what's happening with this one. Check with 3PAO.
- [ ] V14-PL2 system security plan
  - Going to document as operational requirement instead - tracked at https://github.com/18F/cg-product/issues/400

False positives awaiting verification from 3PAO
- [ ] V1-CM-6 version disclosure
   - Ask Veris what the status of this one is
- [ ] V2-CM-6 Kibana directory listing
   - Ask Veris what the status of this one is
- [x] V5-AC6(2) least privilege 
- [x] V6-AC6(10) least privilege
- [ ] V7-AC7 unsuccessful logon attempts
  - Likely need to document as operational requirement (tracked in SAR, [drafting here](https://docs.google.com/document/d/1sMVT-lcgHzzNvsvxqpEJ3OY04WXwAQhY5ObA0gIPrzg/edit), associated issue is https://github.com/18F/cg-product/issues/414)
- [ ] V8-AC22 publicly accessible content
  - Need to set up training system (tracked in SAR); associated issues are https://github.com/18F/cg-product/issues/414 and https://github.com/18F/cg-product/issues/546
- [ ] V13-SC12(2) cryptographic key establishment and management
  - Waiting on JAB opinion
- [x] V15-SI4(23) monitoring

Things that need to be done to close all of these:
- [x] Veris to send updated SAP
- [x] Put deadlines on the SSP chunks and let Bridget know them
- [ ] Do the undone things
- [ ] Prompt Veris to test remaining documentation work
- [ ] Prompt Veris to test remaining technical work
- [x] Veris to send updated SAR
- [ ] Unknown set of things that Veris will want to test

**Comment by badorward** [comment]
- V2 is not listed as a pending FP in the POA&M submitted on 11/1.
- V14 and V73 are listed as pending FPs in the POA&M.
- Code-scanning (SI-3) - did you actually implement something new/different to meet this requirement (Code Climate)?  This may need to be verified.
- Need Veris to confirm testing of operator authentication via SecureAuth.

**Comment by mogul** [comment]
Much of the above was incorporated into the checklist in the first post.

Veris is uploading screenshots of config settings for us to verify we're matching in subsequent scans.

@badorward mentioned more vulns showing in the latest OWASP scan than are in the corresponding POA&M. It sounds like these are likely ones tagged as false-positive in the SAR; @dlapiduz can you make sure they show up that way in the POA&M? Also can you please look at V2, V14, V73 and V19 to make sure they're consistent?

**Comment by badorward** [comment]
Hi,

I confirmed that 6 of the web vulns were identified in the SAR as FPs (my
apologies). There is one low vuln (highlighted in the attached) that is not
listed in the SAR or captured in the POA&M.

Bridget

**Comment by brittag** [comment]
We have the current round of POA&M updating tracked at https://github.com/18F/cg-product/issues/422 - so I'll add these as checklist items over there.

@badorward Unfortunately GitHub silently discards attachments sent with GitHub issue updates. (Sorry for this weirdness!) Would you be able to re-send it to one of us through normal email? We can then put it in our Google Drive. Thank you!

**Comment by brittag** [comment]
Wrote an email about this, but just to summarize for context - I believe the unlisted item is already tracked as part of V17-CM-6 in the POA&M (scheduled completion date 12/18/2016).

**Comment by brittag** [comment]
I went through the checklist, and I believe the only remaining not-otherwise-tracked work here is that we should double-check with Veris that the following ConMon POAM items can be marked closed (they're not listed in our SAR table):

* V12-IA-5(6): Check protection of system authenticators
* V2-CM-6: Kibana directory listing

**Comment by brittag** [comment]
Found V2-CM-6! Asking about V12-IA-5(6).

**Comment by brittag** [comment]
Everything here is done or tracked elsewhere. Yay!
